## Indexing

16S and 18S barcoding experiments were further prepared for sequencing using indexing primers.

Bead Formation
Sample 5 (no waste water in sample 5) should contain rhodomonas about 10e4 cells/ml
sample 6 should contain cells just from waste water (ie. no added rhodomonas)
sample 7 10e4 cells/ml + waste water

Primers used:

| | N701 | N702 | N703 |
| :---: | :---: | :---: | :---: |
| S507 |Sample 5, 16S |Sample 6, 16S |Sample 7, 16S |
| S508 |Sample 5, 18S |Sample 6, 18S |Sample 7, 18S |


 Prepare following PCR mix (4× per sample + 1 extra volume):

| Reagent | 1X | 25X |
| :---: | :---: | :---: |
| H2O | 12,4 µl | 310 µl
| 5x HF buffer | 4 µl | 100 µl |
| dNTP mix (10 mM each) | 0,4 µl | 10 µl |
| Phusion DNA polymerase (2 U/µl) | 0,2 µl | 5 µl |
| i5 primer (10 µM) | 1 µl | 
| i7 primer (10 µM) | 1 µl | 
Reaction | 19 µl | 95 µl |

Pipette 17 µl mastermix, 1 µl of appropriate primers each, and 1 µl template (purified nested PCR product) to 4 PCR tubes per sample

b) Run the following cycling program:

| Step | Temperature (°C) | Time | Number of cycles |
| :---: | :---: | :---: | :---: |
| Beginning temperature | 98 | 30 s | 1 |
| Denaturation | 98 | 10 s | 12 |
| Annealing | 55 | 20 s | 
| Extension | 72 | 20 s | 
| Final extension | 72 | 5 min | 1
| Storing temperature | 4 | Inf | 1

Pool replicate samples, purify with Monarch kit.

DNA concentration was measured by fluorometry (Qubit) and quality evaluated by electrophoresis.

<td valign="top"><img src="../lab_figures/gel1.png" width=400></td>


           1 kB MW marker
	1. 16S sample 5 (rhodo only)
	2. 16S sample 6 (ww only)
	3. 16S sample 7 (rhodo + ww)
	4. 18S sample 5
	5. 18S sample 6
	6. 18S sample 7

Bands look quite good. Something seems to be amplified in the 100 - 200 bp range, but it mainly shows in the lane 5.

The concentration in the same order as above, in ng/µl:

	1. 3,8
	2. 38,0
	3. 14,4
	4. 15,6
	5. 22,2
22,4
